go/libkb/account.go

```diff
@@ -50,7 +50,7 @@ type Account struct {
 	skbKeyring   *SKBKeyringFile
 	secSigKey    GenericKey // cached secret signing key
 	secEncKey    GenericKey // cached secret encryption key
-	lksec        *LKSec     // local key security
+	lksec        *LKSec     // local key security (this member not currently used)
```

Comment:
I kinda would like to keep it for the day when we refactor key unlocking...I added a comment elsewhere in this file about that.  But I understand that it is cruft...